crates/rpc-types/src/lib.rs

```diff
@@ -20,7 +20,5 @@
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 
 mod eth;
-pub mod serde_helpers;
```

Comment:
we want to keep the reexport for convenience, for now

Reply:
> we want to keep the reexport for convenience, for now

sorry but i don't understand really what you means here

Reply:
`pub use alloy_serde as serde_helpers`

this way we don't break anything